lib/invoice_item_repository.rb

```diff
@@ -0,0 +1,58 @@
+require 'csv'
+
+class InvoiceItemRepository
+  def initialize(path)
+    @path = path
+    @rows = CSV.read(@path, headers: true, header_converters: :symbol)
+    @all = all
+  end
+
+  def all
+    result = @rows.map do |row|
+      InvoiceItem.new(row)
+    end
+  end
+
+  def find_by_id(id)
+    result = @all.find do |row|
```

Comment:
I dont believe you need to set this equal to result and then return result, it should output the same without it